backend/business-partner-agent/src/main/java/org/hyperledger/bpa/config/RuntimeConfig.java

```diff
@@ -64,6 +64,15 @@
     @Property(name = "bpa.ux")
     Map<String, Object> ux;
 
+    @Value("${bpa.title}")
```

Comment:
Where do you use the runtime config for i18n in the frontend? It seems you only use the environment.

Reply:
In `main.js` line 106, that is where the runtime config overrides all the defaults.  When I set env vars such as `VUE_APP_TITLE` and `VUE_APP_I18N_LOCALE` in the BPA container those are not picked up by VUE, so the only way to configure is via the runtime config that we have.  I would prefer to just set those env vars, but I couldn't get that working.